documentation/messages/user.registration.transactional.md

```diff
@@ -58,7 +58,7 @@ Accessed by connecting to RabbitMQ server and sending message in the following f
   subscribed: Boolean,
 
   /* Required. Unix timestamp. Example: `1458067430`. */
-  activity_timestamp: String,
+  activity_timestamp: Number,
```

Comment:
Totally agree, Except that is `activity_timestamp` in existing payload, I just fixed an error in data type. Because it's number, not a string.